Exercise_6/ReversePolishNotation.js

```diff
@@ -0,0 +1,43 @@
+function calculate(argument, extraOperators) {
+    if (!argument) throw new Error(argument + " is not allowed");
+    const signs = argument.split(' ');
+    const operators = {
+        "*": (firstValue, secondValue) => firstValue * secondValue,
```

Comment:
Use single quotes.
https://github.com/airbnb/javascript/blob/master/README.md#strings--quotes

Reply:
It is mostly about what to use: double quotes or single quotes. You need to choose only one of them and use everywhere.